src/viam/components/camera/camera.py

```diff
@@ -48,7 +48,10 @@ async def get_image(
 
         ::
 
-            my_camera = Camera.from_robot(robot=robot, name="my_camera")
+           # from viam.media.video import CameraMimeType    
+           # Add the above to imports at top of file!
+        
+            my_camera = Camera.from_robot(robot=machine, name="my_camera")
```

Comment:
If you haven't, make sure to set your editor to trim whitespace at the end of lines

```suggestion

            my_camera = Camera.from_robot(robot=machine, name="my_camera")
```